server/test/unit/server/db/test_manage.py

```diff
@@ -741,16 +754,11 @@ def test_migration_version_cant_be_zero(self):
 
     def test_migration_version_gap(self):
         """
-        Make sure that we require migration versions to be continuous, with no gaps.
+        Make sure that we allow migrations to have version gaps. Previously version gaps were not
+        allowed, and this call would have caused a validation failure. This test makes sure no such
+        failure happens.
         """
-        error_message = 'Migration version 2 is missing in ' +\
-            'unit.server.db.migration_packages.version_gap.'
-        try:
-            models.MigrationPackage(migration_packages.version_gap)
-            self.fail('The MigrationPackage.MissingVersion exception should have been raised, '
-                      'but was not raised.')
-        except models.MigrationPackage.MissingVersion, e:
-            self.assertEquals(str(e), error_message)
+        models.MigrationPackage(migration_packages.version_gap)
```

Comment:
Probably, but since that would be a fundamental change to the way this entire test module works, I think we're better off at this point leaving it the way it is.